nautobot/extras/models/groups.py

```diff
@@ -257,8 +416,11 @@ def clean_filter(self):
         if not isinstance(self.filter, dict):
             raise ValidationError({"filter": "Filter must be a dict"})
 
+        if self.model is None:
+            raise ValidationError({"filter": "Filter requires a `content_type` to be set"})
+
```

Comment:
Do we need `self._set_object_classes` here to be safe?

Reply:
That's already called inside of `self.model` when it it is accessed, and it requires a valid model class as an argument. IOW, accessing `self.model` is enough.